packages/browser-utils/src/metrics/browserMetrics.ts

```diff
@@ -450,6 +451,34 @@ export function _addMeasureSpans(
     attributes['sentry.browser.measure_start_time'] = measureStartTimestamp;
   }
 
+  // https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure#detail
+  if (entry.detail) {
+    // Handle detail as an object
+    if (typeof entry.detail === 'object') {
+      for (const [key, value] of Object.entries(entry.detail)) {
+        if (value && isPrimitive(value)) {
+          attributes[`sentry.browser.measure.detail.${key}`] = value as SpanAttributeValue;
+        } else {
+          try {
+            // This is user defined so we can't guarantee it's serializable
```

Comment:
l: Can we instead just wrap the whole block inside of `if (entry.detail)` in a try-catch, maybe slightly less redundant?

Reply:
If we wrap everything in try catch it means we stop iterating through `Object.entries(entry.detail)`, which means some valid attributes might not get serialized properly.